src/ai/backend/client/cli/admin/acl.py

```diff
@@ -0,0 +1,32 @@
+import sys
+
+from ai.backend.cli.types import ExitCode
+from ai.backend.client.func.acl import _default_list_fields
+from ai.backend.client.session import Session
+
+from ..extensions import pass_ctx_obj
+from ..types import CLIContext
+from . import admin
+
+
+@admin.group()
+def permission():
+    """
+    Permission administration commands.
+    """
+
+
+@permission.command()
+@pass_ctx_obj
+def list(ctx: CLIContext) -> None:
+    """
+    List atomic permissions.
+    (admin privilege required)
```

Comment:
As this command is already a subset of `admin` group I think it's okay to remove this message.
```suggestion
```